src/assets/styles/10-settings/_transition.scss

```diff
@@ -9,4 +9,6 @@
 :root {
   --transition-duration-fast: 0.2s;
   --transition-duration: 0.4s;
+  --transition-timing-function-in: cubic-bezier(0.23, 1, 0.32, 1);
+  --transition-timing-function-out: cubic-bezier(0.55, 0.085, 0.68, 0.53);
```

Comment:
Not sure about setting this at the root... I know it needs to be shared between components, but setting at root seems to indicate it's a global variable, whereas it currently is only used in the Card components (and I suspect we have other transition timing function values declare elsewhere.

Any thoughts?

Reply:
I had similar thoughts as you initially. It makes sense to scope custom props locally where they're clearly meant for for specific components but these underline properties are used in footer nav links too.

How about ...

- For the most part other timing functions are simply 'ease'. This could be added as `--transition-timing-default: ease`
- 'ease-in-out' is used for things related to the header, but not just in the header - could be globally scoped as `--transition-timing-header`
- The above could be renamed more specifically `--transition-timing-underline-in` and `--transition-timing-underline-out`
- Gallery lightbox and logo use specific cubic-beziers so could use locally scoped properties

Reply:
I like that idea, I think using a specific settings file to hold global transitions makes it clear for codebase novices where they should contain any re-used transition styles.

I wonder if locally scoped variables would benefit from being namespaced, e.g. `--component-transition-timing`.

Reply:
Shall I create a ticket for this?

Reply:
> Shall I create a ticket for this?

Yes please

Reply:
> I wonder if locally scoped variables would benefit from being namespaced, e.g. `--component-transition-timing`.

I'm open to this in principle however I think it depends on the context. Definitely we should do it where it makes sense to but it's worth reading the following article to see if there's another way which might complement our codebase more. 

https://css-tricks.com/breaking-css-custom-properties-out-of-root-might-be-a-good-idea/

Reply:
ace, will read